apps/cdk/package.json

```diff
@@ -4,7 +4,6 @@
   "scripts": {
     "build": "tsc && cdk synth",
     "watch": "tsc -w",
-    "test": "jest",
```

Comment:
I'm going to keep this line the same so that way the diff is smaller; I'm overhauling devops and project specs in a different PR to handle these sorts of small things.